### PR TITLE
add Entity Basket

### PR DESCRIPTION
**Pull request by aliaksharbachou**

### CODE REVIEW

File: src/main/java/internship/mbicycle/storify/model/Basket.java [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+
+/**
+ * @author Aliaksandr Harbachou
+ */
+
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Table(name = "basket")
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/model/Basket.java:19]: Do we need to duplicate table name if it equals to default?

File: src/main/java/internship/mbicycle/storify/model/Basket.java [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+            joinColumns = @JoinColumn(
+                    name = "product_id",
+                    referencedColumnName = "id"
+            ),
+            inverseJoinColumns = @JoinColumn(
+                    name = "basket_id",
+                    referencedColumnName = "id"
+            )
+    )
+    private List<Product> productList = new ArrayList<>();
```
Comment (Estrin-MBicycle) [inline src/main/java/internship/mbicycle/storify/model/Basket.java:37]: Are you sure this initialisation will work fine? Pls check it

**Review by Estrin-MBicycle — CHANGES_REQUESTED** [review]

File: src/main/java/internship/mbicycle/storify/model/Basket.java (hunk 1 above)
Comment (aliaksharbachou) [reply src/main/java/internship/mbicycle/storify/model/Basket.java]: Done

File: src/main/java/internship/mbicycle/storify/model/Basket.java (hunk 2 above)
Comment (aliaksharbachou) [reply src/main/java/internship/mbicycle/storify/model/Basket.java]: Done

**Pull request merged**